### DISCUSSION TITLE
It was hard to find a way to apply css styles only to the first page.

### DISCUSSION THREAD

**Discussion by HasseDAC** [marp-team/marp#451] [General]
I am using Marp for VSCode.

I wanted to apply the style only to the first page, as shown below, but was unable to do so.

If I put the pseudo-class `:first-child` in the section tag, the style would be applied to all section tags.
```md
---
marp: true
---

<style>

section:first-child {
  background-color: black;
}

</style>

#Slide 1

```
And using the id `#1` as shown below did not work either.

```md
---
marp: true
---

<style>

section#1 {
  background-color: black;
}

</style>

#Slide 1

```

And finally I found #440.
I could do it with `<style scoped>`.

```md
---
marp: true
---

<style>
  ...
</style>

<style scoped>
  section {
      background-color: black;
  }
</style>

#Slide 1

---

#Slide 2

```

**Comment by yhatt** [comment]
Mark as duplicated with #440. Documentation: https://marpit.marp.app/theme-css?id=scoped-style
